website/fiddle/public/wc/luigiExampleWC.js

```diff
@@ -0,0 +1,265 @@
+import "./f_selectbox.js";
+import "./f_busy.js";
+import "./f_input.js";
+
+const template = document.createElement('template');
+template.innerHTML = `
+
+<!DOCTYPE html>
+<html lang="EN">
+<head>
+    <meta charset="utf-8">
+    <title></title>
+    <link href="//unpkg.com/fundamental-styles@latest/dist/fundamental-styles.css" rel="stylesheet">
+    
+    <style>
+      .fd-page{
+        padding: 20px;
+      }
+    </style>
+    <script>
+        function toggleElAttrs(id, toggleAttrs) {
+            let ref = document.getElementById(id);
+            if (ref && Array.isArray(toggleAttrs) && toggleAttrs.length){
+                for(var i = 0; i < toggleAttrs.length; i++) {
+                    var val = ref.getAttribute(toggleAttrs[i]);
+                    if(val === 'true') {
+                        setElAttr(id, toggleAttrs[i], 'false');
+                    } else if (val === 'false') {
+                        setElAttr(id, toggleAttrs[i], 'true');
+                    }
+                }
+            }
+       }
+    </script>
+</head>
+<body>
+      <main class="fd-page">
+        <header>
+            <div class="fd-container">
+               <div class="fd-row">
+                <div class="fd-col">
+                  <div class="docs-layout-grid-bg docs-layout-grid-bg--color-1">
+                    <h3>Example WC Using Luigi Client, Other imported web Components and Sending/Receiving Data</h3>
+                  </div>
+                </div>
+              </div>
+            </div>
+        </header>
+        
+        <div class="fd-page__content" role="region" style="padding-top:20px;">
+            <div class="fd-container">
+              <div class="fd-row">
+                <div class="fd-col">
+                      <div>Please Select Alert type</div>
+                      <div style="padding-top:5px;">
+                        <div style="width: 200px;">
+                          <fundamental-list-box id="alertBox" placeholder="Select Alert Type" default_option="" option1="info" option2="success" option3="warning" option4="error" /></div>
+                        </div>
+                      </div>
+                </div>
+              </div>
+          </div>
+          <br />
```

Comment:
only some indentation of the last closing div after list box
```suggestion
        <div class="fd-page__content" role="region" style="padding-top:20px;">
            <div class="fd-container">
              <div class="fd-row">
                <div class="fd-col">
                      <div>Please Select Alert type</div>
                      <div style="padding-top:5px;">
                        <div style="width: 200px;">
                          <fundamental-list-box id="alertBox" placeholder="Select Alert Type" default_option="" option1="info" option2="success" option3="warning" option4="error" />
                        </div>
                    </div>
                  </div>
                </div>
              </div>
          </div>
          <br />
```